### PR TITLE
Clean up the changelog

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it adds more detailed changelog related to the content that we added for transactions.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,35 +2,107 @@
...
+    - Add new webhooks:
+      - `TRANSACTION_CHARGE_REQUESTED` - triggered when a staff user request charge for the transaction.
+      - `TRANSACTION_REFUND_REQUESTED` - triggered when a staff user request refund for the transaction.
+      - `TRANSACTION_CANCELATION_REQUESTED` - triggered when a staff user request cancelation for the transaction.
+    - Add new webhook subscriptions:
+      - `TransactionChargeRequested` - Event sent when transaction charge is requested.
+      - `TransactionRefundRequested` - Event sent when transaction refund is requested.
+      - `TransactionCancelationRequested` - Event sent when transaction cancelation is requested.
+    - Add new fields:
+      - `OrderSettings.markAsPaidStrategy` - Determine what strategy will be used to mark the order as paid
```
Comment (kadewu) [inline CHANGELOG.md:24]:
```suggestion
      - `OrderSettings.markAsPaidStrategy` - Determine what strategy will be used to mark the order as paid.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,35 +2,107 @@
...
+        - `TRANSACTION_CHARGE_REQUESTED`
+        - `TRANSACTION_CANCEL_REQUESTED`
+        - `TRANSACTION_MARK_AS_PAID_FAILED`
+    - Add new input fields:
+      - `TransactionCreateInput` & `TransactionUpdateInput`:
+        - `name` - The name of the transaction.
+        - `message` -  The message of the transaction.
+        - `pspReference` - The PSP Reference of the transaction.
+        - `amountCanceled` - Amount canceled by this transaction.
+        - `externalUrl` - The url that will allow to redirect user to payment provider page with transaction
```
Comment (kadewu) [inline CHANGELOG.md:64]:
```suggestion
        - `externalUrl` - The url that will allow to redirect user to payment provider page with transaction.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,35 +2,107 @@
...
+        - `totalCaptured` - Use `totalCharged` instead. Will be removed in Saleor 4.0
+      - `OrderEvent`:
+        - `status` - Use `TransactionEvent` to track the status of `TransactionItem`. This field will be removed in Saleor 3.13 (Preview feature).
+      - `OrderEventsEnum`:
+        - `TRANSACTION_CAPTURE_REQUESTED` - Use `TRANSACTION_CHARGE_REQUESTED` instead. This field will be removed in Saleor 3.13 (Preview feature).
+        - `TRANSACTION_VOID_REQUESTED` - Use `TRANSACTION_CANCEL_REQUESTED` instead. This field will be removed in Saleor 3.13 (Preview feature).
+
+    - Deprecate input fields:
+      - `TransactionCreateInput` & `TransactionUpdateInput`:
+        - `status` - The amounts can be used to define the current status of transactions. This input field will be removed in Saleor 3.13 (Preview feature).
```
Comment (kadewu) [inline CHANGELOG.md:92]: Missing word `instead`? Or I do not understand the description here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,35 +2,107 @@
...
+        - `type` - Use `name` and `message` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `amountVoided` - Use `amountCanceled` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+      - `TransactionEventInput`:
+        - `status` - Status will be calculated by Saleor. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `reference` - Use `pspReference` instead. This input field will be removed in Saleor 3.13 (Preview feature).
+        - `name` - Use `message` instead. This field will be removed in Saleor 3.13 (Preview feature).
+  - Feature preview breaking change:
```
Comment (maarcingebala) [inline CHANGELOG.md:100]: Shouldn't this go to "Breaking Changes" section?

Comment (korycins) [reply CHANGELOG.md:100]: @maarcingebala It depends if `breaking change` should also include a feature preview notes. I wasn't sure, but I can move it there

**Review by kadewu — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**